app/components/record_details/view.rb

```diff
@@ -57,13 +57,18 @@ def region
     end
 
     def trn_row
-      return nil unless [trainee.trn, trainee.submitted_for_trn_at].any?(&:present?)
+      return nil unless [trainee.trn, trainee.submitted_for_trn_at, trainee.hesa_record?].any?(&:present?)
 
       if trainee.trn.present?
         {
           field_label: t(".trn"),
           field_value: trainee.trn,
         }
+      elsif trainee.hesa_record?
```

Comment:
Did we check whether we should be showing this for a missing TRN? HESA trainees should all have TRNs, so it still feels strange for us to be saying "Not provided from HESA update"

Reply:
There are 388 trainees with hesa ID's and no trns. Additionally there are 85 trainees where created from hesa = true and there is no trn.  Also trn is an optional field according to the hesa xml I've found, so I am not sure if we should leave this not provided from Hesa text in. Please see the SQL I used in blazer below to get these figures:

SELECT COUNT(trainees) FROM trainees WHERE hesa_id IS NOT NULL AND trn IS NULL; returns 388
SELECT COUNT(trainees) FROM trainees WHERE created_from_hesa IS true AND trn IS NULL; returns 85

Reply:
If there's no TRN, then they should be in `Pending TRN` state.

Reply:
Ok, removed the code handling the not imported from hesa message from trn